test/k8sT/Policies.go

```diff
@@ -453,6 +455,89 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 				res.ExpectFail("Egress DNS connectivity should be denied for pod %q", pod)
 			}
 		})
+
+		It("Allows traffic with k8s default-allow ingress policy", func() {
+
+			res := kubectl.ExecPodCmd(
+				helpers.DefaultNamespace, appPods[helpers.App2],
+				helpers.CurlFail(fmt.Sprintf("http://%s/public", clusterIP)))
+			res.ExpectSuccess("%q cannot curl clusterIP %q", appPods[helpers.App2], clusterIP)
+
+			res = kubectl.ExecPodCmd(
+				helpers.DefaultNamespace, appPods[helpers.App3],
+				helpers.CurlFail(fmt.Sprintf("http://%s/public", clusterIP)))
+			res.ExpectSuccess("%q cannot curl to %q", appPods[helpers.App3], clusterIP)
+
+			By("Installing ingress default-allow")
+
+			eps := kubectl.CiliumEndpointPolicyVersion(ciliumPod)
+
+			_, err := kubectl.CiliumPolicyAction(
+				helpers.KubeSystemNamespace, allowIngress, helpers.KubectlApply, 300)
+			Expect(err).Should(BeNil(),
+				"L3 allow-ingress Policy cannot be applied in %q namespace", helpers.DefaultNamespace)
+
+			defer func() {
```

Comment:
Please don't use defer in the functions, the cleanup should happens on AfterEach.